test/tests.transform.js

```diff
@@ -47,3 +47,102 @@ describe("_blockHoist nodes", function() {
     assert.deepEqual(names, ["hoistMe", "doNotHoistMe", "oyez"]);
   });
 });
+
+context("functions", function() {
```

Comment:
are those good tests? I mean - they are highly coupled to the internal implementation, shouldnt we just test if the outcome is treeshakeable?